src/state/routing/slice.ts

```diff
@@ -114,6 +119,7 @@ export const routingApi = createApi({
       },
       async queryFn(args, _api, _extraOptions, fetch) {
         let fellBack = false
+        const quoteStartMark = performance.mark(`quote-fetch-start-${Date.now()}`)
```

Comment:
so do we want to measure the time between user typing input -> user seeing quote? im guessing the interface-side measurement of latency should include things like debounce, and in that case the quote-fetch-start should really start on user input change and not the routing/slice fetch

because theoretically just the API fetch metrics can be measured on the backend side right 🤔

Reply:
@tinaszheng this ask specifically is for the API fetch latencies - see [this message](https://uniswapteam.slack.com/archives/C02T729PMQE/p1691681536496569)

i'm separately working on logging for those client side metrics that you mentioned